api/specs/api-v3.0.yaml

```
@@ -0,0 +1,1453 @@
+                distance:
+                  '$ref': '#/components/schemas/distance'
+                driver:
+                  '$ref': '#/components/schemas/driver'
+                passenger:
+                  '$ref': '#/components/schemas/passenger'
+      responses:
+        '201':
+          description: OK. Le trajet a bien été enregistré.
+          content:
+            'application/json':
+               schema:
```

yamllint (1.35.1)

[warning] wrong indentation: expected 14 but found 15

(indentation)